Plugins/Flow.Launcher.Plugin.Explorer/Search/ResultManager.cs

```diff
@@ -26,7 +28,7 @@ internal static Result CreateFolderResult(string title, string subtitle, string
                 TitleHighlightData = StringMatcher.FuzzySearch(query.Search, title).MatchData,
                 Action = c =>
                 {
-                    if (c.SpecialKeyState.CtrlPressed)
+                    if (c.SpecialKeyState.CtrlPressed || !(Settings.EnabledPathSearchKeyword || Settings.EnableSearchActionKeyword))
```

Comment:
Is this saying both Settings.EnabledPathSearchKeyword and Settings.EnableSearchActionKeyword evaluate to false? Can we not just do (!Settings.EnabledPathSearchKeyword && !Settings.EnableSearchActionKeyword) ?

Reply:
If we change it to Enabled, what about move Enabled to the end of the name, like PathSearchKeywordEnabled?

Reply:
> Is this saying both Settings.EnabledPathSearchKeyword and Settings.EnableSearchActionKeyword evaluate to false? Can we not just do (!Settings.EnabledPathSearchKeyword && !Settings.EnabledSearchActionKeyword) ?

That will have two !🤣, but it seems more clear. I will change it.

Reply:
> If we change it to Enabled, what about move Enabled to the end of the name, like PathSearchKeywordEnabled?

Sorry for the late reply, I am ok moving Enabled to the end if you want to